pandas/core/frame.py

```diff
@@ -6905,6 +6907,8 @@ def explode(self, column: Union[str, Tuple]) -> "DataFrame":
         ----------
         column : str or tuple
             Column to explode.
+        ignore_index : bool, default False
+            If True, the resulting index will be labeled 0, 1, …, n - 1.
```

Comment:
can you add a versionadded 1.1 tag (is this doc-string similar to other uses of ignore_index)?

Reply:
Done, as much as possible. Right now it is not consistent over other methods. Most use the same description though.

Reply:
For example, `Series.append` uses:

```
        ignore_index : bool, default False
            If True, do not use the index labels.
```
While `Series.sort_values` and `Series.sort_index` use the same description as added to `explode`